### PR TITLE
update request.js

### PR DESCRIPTION
**Pull request by makeco**
coverage `payload.length` to string

### CODE REVIEW

**Comment by makeco** [comment]
ok

File: test/test.js [hunk 1]
```diff
@@ -475,6 +475,18 @@ test('can override stream payload content-length header', (t) => {
   })
 })
 
+test('can override stream payload content-length header without request content-length', (t) => {
+  t.plan(1)
+  const dispatch = function (req, res) {
+    res.writeHead(200, { 'Content-Type': 'text/plain' })
+    res.end(req.headers['content-length'])
```
Comment (mcollina) [inline test/test.js]: You should do an assertion here rather than returning it.

Comment (makeco) [reply test/test.js]:
```js
const dispatch = function (req, res) {
    res.writeHead(200, { 'Content-Type': 'text/plain' })
    t.equal(req.headers['content-length'], '2')
}
```

File: test/test.js [hunk 2]
```diff
@@ -475,6 +475,18 @@ test('can override stream payload content-length header', (t) => {
...
+  }
+
+  inject(dispatch, { method: 'post', url: '/', payload: getTestStream() }, (res) => {
```
Comment (mcollina) [inline test/test.js]: also you should check here if the `content-length` is a number.

**Comment by delvedor** [comment]
Before accept this, I want to answer the question posted in https://github.com/fastify/fastify/pull/467#issuecomment-344602561.

**Pull request merged**